package.json

```diff
@@ -5,18 +5,15 @@
     "@fortawesome/fontawesome-free": "^5.7.2",
     "@github/markdown-toolbar-element": "1.2.0",
     "jquery": "3.5.1",
-    "popper.js": "^1.14.3",
     "bootstrap": "^4.3.1",
-    "bootswatch": "^4.4.1",
-    "datatables.net-bs4": "^1.10.18",
-    "datatables.net-buttons-bs4": "^1.5.4"
```

Comment:
These dependencies are removed as we're already pulling them in from other sources.